documentation/src/main/asciidoc/user_guide/chapter-76-Simple_Cache.adoc

```diff
@@ -0,0 +1,47 @@
+== Simple Cache
+
+Traditional local caches use the same architecture as clustered caches - interceptor stack. That way a lot of the functionality can be reused. However, if the advanced features are not needed and the performance is more important, the interceptor stack can be stripped away and simple cache can be used.
```

Comment:
s/functionality can be reused/implementation can be reused/ ?

Also, for easy maintenance it's recommended to wrap lines and to start each sentence on a new line.